pkgs/development/libraries/libuv/default.nix

```diff
@@ -26,6 +26,8 @@ stdenv.mkDerivation (finalAttrs: {
   version = "1.46.0";
   pname = "libuv";
 
+  __structuredAttrs = true;
```

Comment:
The package has to be checked and diffed on all supported OS'es when enabling structuredAttrs because there may be hooks and platform specific hooks which don't work with it.

Alternative is to put `disabledTests` in to `passthru`

Reply:
thanks for the precision but how is a simple contributor like myself supposed to check that ? is there some kind of tooling or anyone to ping who can confirm it works on all platforms ?

Reply:
I don't know, im hesitant to set `__structuredAttrs` because only the core kinda works with it.

Reply:
it's sad we can't use __structuredAttrs after all these years. passthru would be fine here but I would prefer if we could enable __structuredAttrs. So maybe let's wait for after the release and then:
- see if people with the knowhow can actually confirm it's ok
- alternatively merge and fix whatever needs fixing or revert if too painful 

I've enabled _structuredAttrs in the lua ecosystem without anyone complaining so I am tempted to think __structuredAttrs  could work here too without problem.

Reply:
There's also the point that this list doesn't or shouldn't be a part of the drv since it's referred to directly via finalAttrs